core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts

```diff
@@ -481,17 +482,12 @@ export class ConversationSkinComponent {
       }
 
       if (this.diagnosticTestTopicTrackerModel) {
-        this.enableFeedback = false;
-        this.allowOnlySingleAttemptForAnswering = true;
-        this.showOnlyLastInputPairResponse = true;
-        this.enableNavigationThroughCardHistory = false;
-        this.enableCheckpointCelebrationModal = false;
-      } else {
-        this.enableFeedback = true;
-        this.allowOnlySingleAttemptForAnswering = false;
-        this.showOnlyLastInputPairResponse = false;
-        this.enableNavigationThroughCardHistory = true;
-        this.enableCheckpointCelebrationModal = true;
+        this.feedbackIsEnabled = false;
+        this.learnerCanOnlyAttemptQuestionOnce = true;
+        this.lastInputOutputPairIsOnlyShown = true;
```

Comment:
Should be onlyLastInputResponsePairIsShown -- what you have here means something different (it suggests that this is controlling whether the only thing we do with the last input output pair is to show it, and whether or not to do other things with it).

If you prefer, you can flip it and say something like "inputOutputHistoryIsShown" -- that might actually be clearer.